packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -5778,6 +5857,11 @@ class SemanticsConfiguration {
       _controlsNodes = <String>{..._controlsNodes!, ...child._controlsNodes!};
     }
 
+    if (_validationResult == SemanticsValidationResult.none &&
+        child._validationResult != _validationResult) {
```

Comment:
should we do some kind of prioritize invalid similar to the getSemanticsData

Reply:
I'm not even sure screen readers would support nested validation results. So maybe we should assert and throw if conflicting validation results are being merged?

Reply:
probably shouldn't throw. we can either generate separate node or have some merging strategy.

Reply:
Done. Merged with invalid taking precedence.